addons/calendar/data/mail_template_data.xml

```diff
@@ -90,7 +90,7 @@
                 </t>
                 <t t-if="object.event_id.location">
                     <li>Location: <t t-out="object.event_id.location or ''">Bruxelles</t>
-                        (<a target="_blank" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">View Map</a>)
+                        <a target="_blank" t-if="is_online and object.event_id.location != object.event_id.appointment_type_id.location_id.name" t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.event_id.location}}">(View Map)</a>
```

Comment:
Why adding the is_online condition here ? Do we only want the map link for appointment ?
IMO we could just do `object.event_id.location != object.event_id.appointment_type_id.location_id.name` that way we still have the link for calendar event (without appointment type) like before.
(same for the other "View Map" below)

Reply:
Removed the `is_online` condition... :+1:

Reply:
I did apply this exact condition earlier as well. However, it leads to some failing tests with the following traceback:
`AttributeError: 'calendar.event' object has no attribute 'appointment_type_id'`
Hence I ensured that the aforementioned condition was checked only if `is_online` was true.

Do you have anything else in mind here?

Reply:
Ha indeed (I forgot that with calendar only, appointment_type_id does not exist)
Then perhaps we could do something like this to avoid the `appointment_type_id` check when we don't need it
`not is_online or object.event_id.location != object.event_id.appointment_type_id.location_id.name`

Reply:
Yes, thought so! I'll apply this change then. Thanks!